cypress/integration/userCanChangeUILanguage.feature.js

```diff
@@ -0,0 +1,18 @@
+describe('user can change UI language', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  context('choosing Swedish language', () => {
+    it('is expected to translate UI into Swedish', () => {
+      cy.get('[data-cy=language-dropdown]').click();
+      cy.get('.visible.menu.transition').contains('se').click();
+      cy.get('[data-cy=home-tab]').should('contain', 'Hem');
+      cy.get('[data-cy=backyard-tab]').should('contain', 'Bakgard');
```

Comment:
Bakgard here, shouldn't it be bakgård or does cypress now pick up the å character?

Reply:
It didn't take it somehow, read a lot everywhere and all we do is right, we have to ask Thomas maybe.